core/src/test/java/org/elasticsearch/index/fielddata/BinaryDVFieldDataTests.java

```diff
@@ -56,24 +58,25 @@ public void testDocValue() throws Exception {
         bytesList1.add(randomBytes());
         bytesList1.add(randomBytes());
         XContentBuilder doc = XContentFactory.jsonBuilder().startObject().startArray("field").value(bytesList1.get(0)).value(bytesList1.get(1)).endArray().endObject();
-        ParsedDocument d = mapper.parse("test", "test", "1", doc.bytes());
+        ParsedDocument d = mapper.parse(SourceToParse.source("test", "test", "1", doc.bytes(), 
```

Comment:
Sometimes it improves readability to of split lines to always have at least 2 "things" on the next line. In this case I'd have moved `doc.bytes()` to the next line. You certainly don't have to.